docs/operations/snapshot_restore_api.md

```diff
@@ -0,0 +1,94 @@
+# Snapshot Restore API
+
+The `snapshot.kubevirt.io` API Group defines resources for snapshotting and restoring KubeVirt `VirtualMachines`
+
+## Prerequesites
+
+### VolumeSnapshotClass
+
+KubeVirt leverages the `VolumeSnapshot` functionality of Kubernetes [CSI drivers](https://kubernetes-csi.github.io/docs/drivers.html) for capturing persistent `VirtualMachine` state.  So, you should make sure that your `VirtuallMachine` uses `DataVolumes` or `PersistentVolumeClaims` backed by a `StorageClass` that supports `VolumeSnapshots` and a `VolumeSnapshotClass` is properly configured for that `StorageClass`.
```

Comment:
second instance of VirtualMachine is mispelled with two L (els).

Reply:
Fixed